corehq/messaging/smsbackends/turn/views.py

```diff
@@ -13,21 +14,23 @@ def backend_class(self):
         return SQLTurnWhatsAppBackend
 
     def post(self, request, api_key, *args, **kwargs):
-        request_body = json.loads(request.body)
-        for message in request_body.get('messages', []):
-            message_id = message.get('id')
-            from_ = message.get('from')
-            if message.get('type') == 'text':
-                body = message.get('text', {}).get('body')
-            elif message.get('type') == 'image':
-                body = message.get('image', {}).get('caption')  # this doesn't seem to work yet
+        if TURN_IO_BACKEND.enabled_for_request(request):
```

Comment:
It seems odd to me that we would just ignore incoming messages if this FF get's disabled.

Reply:
Yeah, I also thought so, but my reasoning was that if a user turns off the FF, he/she should not expect to still have the functionality of that FF (there's a discussion regarding FF's and their use after disabling them on Slack in `#web-team`; based on that outcome this code might change along with a few other places).